app/features/TransactionSlice.ts

```diff
@@ -717,4 +717,28 @@ export const loadingTransactionsSelector = (state: RootState) =>
 export const hasMoreTransactionsSelector = (state: RootState) =>
     state.transactions.hasMore;
 
+const isPending = (t: TransferTransaction) =>
+    t.status === TransactionStatus.Pending;
+
+export const hasPendingTransactionSelector = (
```

Comment:
I think we have a database call hasPending, for a specific account. TODO: Check this again when I'm done reviewing